layouts/default.html

```diff
@@ -21,6 +21,9 @@
                 <li><%= link_to "Tags", '/tags/' %></li>
                 <li><%= link_to "Atom", '/atom/' %></li>
                 <li><%= link_to "XML Sitemap", '/sitemap/' %></li>
+                <li><a href='mailto:orga[at]unfug(dot)org'>Mail us</a></li>
```

Comment:
The point is that automatic page crawlers are not able to send spam, but a human can easily fix this. I'd like to keep it this way.

What do you think, @fkemmer ?